relay-server/src/services/store.rs

```diff
@@ -339,22 +345,27 @@ impl StoreService {
         let mut dropped = SourceQuantities::default();
         let mut error = None;
 
+        let encodings = self.global_config.current().options.metric_bucket_encodings;
+
         for bucket in buckets {
             // Create a local bucket view to avoid splitting buckets unnecessarily. Since we produce
             // each bucket separately, we only need to split buckets that exceed the size, but not
             // batches.
             for view in BucketsView::new(&[bucket]).by_size(batch_size).flatten() {
-                let message = MetricKafkaMessage {
-                    org_id: scoping.organization_id,
-                    project_id: scoping.project_id,
-                    name: view.name(),
-                    value: view.value(),
-                    timestamp: view.timestamp(),
-                    tags: view.tags(),
-                    retention_days: retention,
-                };
+                let namespace = MetricResourceIdentifier::parse(view.name())
+                    .map(|mri| mri.namespace)
+                    .unwrap_or(MetricNamespace::Unsupported);
```

Comment:
Do we actually wan to create a metric message if we cannot parse the MRI? Or should we skip it instead?

Reply:
This will already skip unsupported namespaces in the store function. I opted not to make the decision here to skip so it is only in a single place (would be 3 otherwise).